.github/workflows/c8run-release.yaml

```diff
@@ -54,40 +95,66 @@ jobs:
           secrets: |
             secret/data/products/distribution/ci NEXUS_USERNAME;
             secret/data/products/distribution/ci NEXUS_PASSWORD;
-
       - uses: actions/setup-go@v5
         with:
           go-version: '>=1.23.1'
           cache: false  # disabling since not working anyways without a cache-dependency-path specified
-
       - name: Build c8run
         run: go build
         working-directory: ./c8run
-
       - name: make a package
         run: ./c8run package
         shell: bash
         working-directory: ./c8run
         env:
           JAVA_ARTIFACTS_USER: ${{ steps.secrets.outputs.NEXUS_USERNAME }}
           JAVA_ARTIFACTS_PASSWORD: ${{ steps.secrets.outputs.NEXUS_PASSWORD }}
-
-      - name: upload asset
+      # - name: Clean up old release
+      #   env:
+      #     GH_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+      #   run: |
+      #     gh release view ${{ env.CAMUNDA_RUN_NAME }} &&
+      #       gh release delete ${{ env.CAMUNDA_RUN_NAME }} --cleanup-tag ||
+      #       true
+      #     # To avoid race condition when the release is created again.
+      #     sleep 5
+      # - name: Create new release
+      #   run: |
+      #     gh release create ${{ env.CAMUNDA_RUN_NAME }} --target ${{ inputs.branch }} --notes "${{ env.CAMUNDA_RUN_NAME }}"
+      # env:
+      #   GH_TOKEN: ${{ github.token }}
```

Comment:
I'm thinking of a way this could be an option. If you set inputs.cleanUpOldRelease == true, then these steps will be triggered.  I think this would be better than commented out steps.  The complication with that is we would only want one job to actually run those steps. so perhaps a separate job for just this action.

Reply:
I made some changes but then realized then weren't pushed :grin: